server/lib/update-fields.js

```diff
@@ -0,0 +1,40 @@
+import Customer from '../models/customer'
+import Volunteer from '../models/volunteer'
+
+export default {
```

Comment:
I'd export each function separately so you can `import {updateVolunteer} from ...` and just use `updateVolunteer` instead of `update.updateVolunteer`